data/decks/web.yaml

```diff
@@ -1,6 +1,12 @@
 ---
-name: Web test deck
-description: This deck runs HTTP Header Field Manipulation and the Web Connectivity test
+name: Test URLs
+description: "This deck examines whether (and how) websites are blocked and
+tries to detect censorship and/or surveillance systems in your network. The Web
+Connectivity test examines whether access to websites is blocked through DNS
+tampering, TCP/IP blocking, or by a transparent HTTP proxy. The
+HTTP Header Field Manipulation test tries to detect censorship and/or
```

Comment:
Actually there is no such thing as "category", what has changed is only the deck name (that is to say there is no way to put multiple decks inside of a certain "category").

I think maybe the best thing to do is to rename 'Test URLs' to 'Web tests'.